.github/workflows/build-project.yml

```diff
@@ -47,6 +47,9 @@ jobs:
           use-github-cache: false
           use-mathlib-cache: true
 
+      - name: check that Project.lean is up to date
```

Comment:
I tend to prefer beginning the job step name with capital letters. Could you please turn "check" in "Check"? 

Thanks.

Reply:
```suggestion
      - name: Check that Project.lean is up to date